js/main.js

```diff
@@ -0,0 +1,62 @@
+// the book collection
```

Comment:
- [ ] Nice job so far 👏. Your website is working well except for one thing. When I reload the page it is not showing the books or not getting the books from the local storage. Please kindly fix this issue 🍀👍

Reply:
Done 👍